src/database/food.go

```diff
@@ -0,0 +1,34 @@
+package database
+
+import (
+	"foodloop/src/models"
+)
+
+func GetFoods(foodlistID string) ([]models.Food, error) {
```

Comment:
there's alr a method `GetFoodlist` that returns foods of each foodlist

eg)

{
  "foodlistID": 3,
  "foodlistName": "biryani",
  "foodlistTime": "0000-01-01T18:00:00Z",
  "foodlistDay": "Wednesday",
  "foodlistCurrIdx": 1,
  "Foodlist": [
    {
      "foodID": 61,
      "foodName": "Hyderabadi Biryani",
      "descriptions": "Traditional Hyderabadi biryani made with fragrant basmati rice, tender pieces of marinated meat, and aromatic spices.",
      "category": "burger",
      "foodIndex": 1
    },
    {
      "foodID": 62,
      "foodName": "Lucknowi Biryani",
      "descriptions": "Lucknowi biryani featuring long-grain rice cooked with succulent pieces of meat, saffron-infused milk, and fried onions.",
      "category": "burger",
      "foodIndex": 2
    },
   ...